avocado/core/settings.py

```diff
@@ -107,6 +107,8 @@ def convert_value_type(value, value_type):
             value_type = float
         elif value_type == 'list':
             value_type = list
+        elif value_type == 'path':
+            value_type = os.path.expanduser
```

Comment:
About the name change, it should not be changed in this patch, but I think it'd be clearer to change it before this patch (or after, but before would be IMO better). Anyway it's not strong opinion, only the `value_type` does not fit in my eyes. It's not type, it's really a conversion function called later in the code.